Cargo.toml

```diff
@@ -32,6 +32,7 @@ serde_derive = "1.0"
 serde_json = "1.0"
 structopt = "0.3"
 tokio = "0.2.20"
+lazy_static = "1.4.0"
```

Comment:
I would recommend using [`once_cell`](https://docs.rs/once_cell/1.4.1/once_cell/) crate, because it is going to be a standard way to handle lazy initialization of static variables. By "standard" I mean literally standard, this crate is already exposed as part of `std`, though unstable: https://github.com/rust-lang/rust/issues/74465